python/adelphi/adelphi/anonymize.py

```diff
@@ -38,59 +33,70 @@ def get_name(original_name, prefix):
 def anonymize_keyspace(keyspace):
     keyspace.name = get_name(keyspace.name, KEYSPACE_PREFIX)
 
+    # anonymize udts at once because they can reference
+    # one another
+    udts = keyspace.user_types.values()
+    anonymize_udts(udts)
+
     for table in keyspace.tables.values():
         anonymize_table(table)
 
-    for udt in keyspace.user_types.values():
-        anonymize_udt(udt)
+    # remove functions, aggregates and views for now
+    keyspace.functions = {}
+    keyspace.aggregates = {}
+    keyspace.views = {}
```

Comment:
Is the removal of all of these required for anonymization?  Seems like we're giving up a fair amount of useful schema info here.

Reply:
@absurdfarce do you think we can safely anonymize functions and aggregates?

Maybe materialized views could be added, but since views aren't recommended for prod, it seemed like a lower priority that shouldn't block the rest of the PR. We can open a ticket to track this though.

Reply:
My inclination is to say we probably can do this, but yeah, it can be held off for future work.  It does seem like including as many of these as possible in some capacity is probably worth it, though, since it will make for more robust validation testing.

Reply:
> My inclination is to say we probably can do this

Did you consider anonymizing the function body across all the supported languages? And even so, I think hiding the implementation of an UDF algorithm is more important than just anonymizing its variable names.
Anyway, that's a much larger scope and we can revisit it the future as you said. +1